modules/ROOT/pages/index-self-managed.adoc

```diff
@@ -108,6 +108,14 @@ Runtime Fabric on Self-Managed Kubernetes does not include external log forwardi
 
 For Titanium customers, Runtime Fabric on Self-Managed Kubernetes supports logging using Anypoint Monitoring. See xref:monitoring::logs.adoc[Logs in Anypoint Monitoring] for more information.
 
+=== Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides basic metrics for applications and API gateways deployed to Runtime Fabric. 
+
+To collect metrics, Anypoint Monitoring sidecars run in all pods deployed to Runtime Fabric. See xref:manage-monitor-applications.adoc[Monitor Applications Deployed to Runtime Fabric] for details, including how to enable or disable monitoring. 
+
+Runtime Fabric does not support third-party monitoring solutions.
```

Comment:
I don't think we need to say this. Some monitoring solutions can inject themselves into running pods, for example https://www.dynatrace.com/support/help/setup-and-configuration/setup-on-container-platforms/kubernetes/deploy-oneagent-on-kubernetes-for-application-only-monitoring/.

The truth is that we do not give customers the ability to configure any specific monitoring solution from RTF itself. We do not have integration with specific vendors like Datadog or NewRelic etc. 

How about something softer eg  _"does not _provide_ support for integrating third-party monitoring solutions"_ ?

@vli63127

Reply:
I like that suggestion